packages/react/use-previous/src/usePrevious.tsx

```diff
@@ -3,15 +3,18 @@ import * as React from 'react';
 function usePrevious<T>(value: T) {
   // The ref object is a generic container whose current property is mutable ...
   // ... and can hold any value, similar to an instance property on a class
```

Comment:
Oh, one little favour while we're here if you don't mind... Can we remove this comment? 🙏 I've never understood why we have a comment explaining how refs work 😅

Reply:
Smells like copy/paste 😀